specs/Media.md

```diff
@@ -0,0 +1,146 @@
+interface ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler;
+
+[uuid(76eceacb-0462-4d94-ac83-423a6793775e), object, pointer_default(unique)]
+interface ICoreWebView2_2 : ICoreWebView2 {
+  /// Adds an event handler for the `IsMutedChanged` event.
+  /// `IsMutedChanged` runs when the mute state changes. The event may run 
+  /// when `Mute` and `Unmute` are called.
+  HRESULT add_IsMutedChanged(
+      [in] ICoreWebView2StagingIsMutedChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with `add_IsMutedChanged`.
+  HRESULT remove_IsMutedChanged(
+      [in] EventRegistrationToken token);
+
+  /// Mutes all audio output from this CoreWebView2.
+  HRESULT Mute();
+  /// Unmutes all audio output from this CoreWebView2.
+  HRESULT Unmute();
+  /// Indicates whether all audio output from this CoreWebView2 is muted or not.
+  [propget] HRESULT IsMuted([out, retval] BOOL* isMuted);
+  
+  /// Adds an event handler for the `IsCurrentlyAudibleChanged` event.
+  /// `IsCurrentlyAudibleChanged` runs when the audible state changes.
+  HRESULT add_IsCurrentlyAudibleChanged(
+      [in] ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with `add_IsCurrentlyAudibleChanged`.
+  HRESULT remove_IsCurrentlyAudibleChanged(
+      [in] EventRegistrationToken token);
+  
+  /// Indicates whether any audio output from this CoreWebView2 is audible.
+  /// CoreWebView2 can be audible when muted, IsCurrentlyAudible is used to indicate
+  /// if there are audio/media currently playing.
+  [propget] HRESULT IsCurrentlyAudible([out, retval] BOOL* isAudible);
+}
+
+/// Implements the interface to receive `IsMutedChanged` events.  Use the
+/// IsMuted method to get the mute state.
+[uuid(B357DC3B-D4C3-4FDE-BF45-C11ECE606B98), object, pointer_default(unique)]
+interface ICoreWebView2StagingIsMutedChangedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.  No event args exist
+  /// and the `args` parameter is set to `null`.
+  HRESULT Invoke([in] ICoreWebView2Staging2* sender, [in] IUnknown* args);
+}
+
+/// Implements the interface to receive `IsCurrentlyAudibleChanged` events.  Use the
+/// IsCurrentlyAudible method to get the audible state.
+[uuid(5DEF109A-2F4B-49FA-B7F6-11C39E513328), object, pointer_default(unique)]
+interface ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.  No event args exist
+  /// and the `args` parameter is set to `null`.
+  HRESULT Invoke([in] ICoreWebView2Staging2* sender, [in] IUnknown* args);
+}
+```
+
+## .NET and WinRT
+
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+    public partial class CoreWebView2
+    {
+        // There are other API in this interface that we are not showing 
```

Comment:
Missing the events

Reply:
Seems like eventHandlers are not included in the Microsoft.Web.WebView2.CoreWebView2.idl. Should I still add it?

Reply:
The events declaration should be included and the event args (if there are any for an event) but winrt & .net don't need to define the event handler interface

Reply:
There is no event args.